24_clustering.Rmd

```diff
@@ -33,7 +33,10 @@ library(bluster)
 tse <- transformCounts(tse, method = "relabundance")
 ```
 
-The clustering can be done on features or samples.
+*bluster*'s main function `clusterRows`, used to do the clustering can be 
+used via mia's `cluster` function.
```

Comment:
Reconsider how much to explain function names in the text. They will come in the code examples in any case. There are no strict rules on this but the main text should be fluent and easy to read, and many function names make it more technical and we want to go beyond a technical manual in this online book.